src/wooting-usb.c

```diff
@@ -46,8 +44,50 @@ static uint16_t getCrc16ccitt(const uint8_t* buffer, uint16_t size)
 	return crc;
 }
 
-void wooting_usb_disconnect(bool trigger_cb) {
+#pragma region Set Meta
+
+void reset_meta() {
```

Comment:
These functions are only local right? We can add the static keyboard to make them local, in [C function are global by default](https://www.geeksforgeeks.org/what-are-static-functions-in-c/).

Reply:
The definition of `wooting_usb_meta`  can also be static to indicate it's only used in this module.

Reply:
Yeah, they're just local. I was under the impression that you could only access functions that are defined in the header, but I guess that's not the case. I'll change them over to static